### PR TITLE
Added doods component documentation

### PR DESCRIPTION
**Pull request by snowzach**
**Description:**
Add support for DOODS Object Detection Service

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant/pull/26208

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10228"></a>

### CODE REVIEW

File: source/_components/doods.markdown [hunk 1]
```diff
@@ -0,0 +1,160 @@
+---
+title: "DOODS"
+description: "Detect and recognize objects with DOODS."
+logo: home-assistant.png
+ha_category:
+  - Image Processing
+ha_iot_class: Local Polling
+ha_release: 0.82
```
Comment (klaasnicolaas) [inline source/_components/doods.markdown]:
```suggestion
ha_release: 0.99
```

File: source/_components/doods.markdown [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/doods.markdown]: Please remove the `redirect_from`

File: source/_components/doods.markdown [hunk 3]
```diff
@@ -0,0 +1,160 @@
+---
+title: "DOODS"
+description: "Detect and recognize objects with DOODS."
+logo: home-assistant.png
```
Comment (klaasnicolaas) [inline source/_components/doods.markdown]: The Home Assistant logo can only be used for own integrations, so please remove the line if there is no logo.

File: source/_components/doods.markdown [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+You need to have DOODS running somewhere. It's easiest to run as a docker container and deployment is described on docker hub 
+[DOODS - Docker](https://hub.docker.com/r/snowzach/doods)
+
+## Configuration
+
+The configuration loosely follows the tensorflow configuration. To enable this platform in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+image_processing:
```
Comment (klaasnicolaas) [inline source/_components/doods.markdown:23]: Only required config variables in the first example.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/doods.markdown (hunk 3 above)
Comment (snowzach) [reply source/_components/doods.markdown]: Updated.

File: source/_components/doods.markdown (hunk 4 above)
Comment (snowzach) [reply source/_components/doods.markdown:23]: @klaasnicolaas removed optional items.

**Comment by klaasnicolaas** [comment]
just remove the `redirect_from` and then we can further review your PR.

File: source/_components/doods.markdown [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+ha_release: 0.99
+redirect_from:
+ - /components/image_processing.doods/
+---
+
+The `doods` image processing platform allows you to detect and recognize objects in a camera image using [DOODS](https://github.com/snowzach/doods/). The state of the entity is the number of objects detected, and recognized objects are listed in the `summary` attribute along with quantity. The `matches` attribute provides the confidence `score` for recognition and the bounding `box` of the object for each detection category.
+
+## Setup
+
+You need to have DOODS running somewhere. It's easiest to run as a docker container and deployment is described on docker hub
```
Comment (klaasnicolaas) [inline source/_components/doods.markdown:14]: Would be great if you could make an add-on of this docker container so that it is easy to set up in HA.

**Comment by snowzach** [comment]
> 
> 
> just remove the `redirect_from` and then we can further review your PR.

Sure, that's done.

File: source/_components/doods.markdown (hunk 5 above)
Comment (snowzach) [reply source/_components/doods.markdown:14]: Once I get the component accepted, I'll work on that.

File: source/_components/doods.markdown (hunk 2 above)
Comment (snowzach) [reply source/_components/doods.markdown]: Done

**Review by klaasnicolaas — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_components/doods.markdown [hunk 6]
```diff
@@ -0,0 +1,151 @@
+---
+title: "DOODS"
+description: "Detect and recognize objects with DOODS."
+ha_category:
+  - Image Processing
+ha_iot_class: Local Polling
+ha_release: 0.99
```
Comment (MartinHjelmare) [inline source/_components/doods.markdown:7]: 0.100

**Comment by arsaboo** [comment]
Merging this as the parent is merged...thanks @snowzach 🎉

**Comment by MartinHjelmare** [comment]
Can someone address the `ha_release` change needed?

**Pull request merged**